src/tr1/game/inject.c

```diff
@@ -499,8 +499,9 @@ static void M_AnimData(INJECTION *injection, LEVEL_INFO *level_info)
         level_info->anim_change_count, inj_info->anim_change_count, fp);
     Level_ReadAnimRanges(
         level_info->anim_range_count, inj_info->anim_range_count, fp);
-    Level_ReadAnimCommands(
-        level_info->anim_command_count, inj_info->anim_cmd_count, fp);
+    VFile_Read(
+        fp, level_info->anim_command_data + level_info->anim_command_count,
+        inj_info->anim_cmd_count * sizeof(int16_t));
```

Comment:
Maybe we could create a set of functions in the Level module, similar to what we do for frames? Even if the only thing they'd do would be to `Memory_Alloc`, `VFile_Read`, and `Memory_Free`.

Reply:
Good call, shifted now to the common level module.